vowpalwabbit/cb.cc

```diff
@@ -17,6 +17,15 @@ using namespace VW::LEARNER;
 
 namespace CB
 {
+std::pair<bool, cb_class> get_observed_cost_cb(const label& ld)
```

Comment:
maybe this should also be renamed to `get_observed_cost_or_default_cb` to match the `adf` case

Reply:
The contract of this function is if the first value is false that the second value should not be read which is why it is named like that compared to the other one